### DISCUSSION TITLE
Problem with config

### DISCUSSION THREAD

**Discussion by Tysonka** [pufferpanel/pufferpanel#1308] [Q&A] [answered]
Hi! 
I installed pufferpanel container to my synology nas for a minecraft server, but I think something went wrong. There is no configuration labels on the left side. What could be the problem?

![Képernyőkép 2024-03-03 021757](https://github.com/pufferpanel/pufferpanel/assets/89782074/1062e50d-4199-43d5-8ab3-72b1878ea808)

**Comment by LordRalex** [comment, marked as answer]
This means the account isn't an admin.
Use `pufferpanel user edit` inside your container to change your account to an admin.
